Content.Client/Stylesheets/StyleNano.cs

```diff
@@ -1455,8 +1455,13 @@ public StyleNano(IResourceCache resCache) : base(resCache)
 
                 Element<TextureButton>().Class("CrossButtonRed").Pseudo(TextureButton.StylePseudoClassHover)
                     .Prop(Control.StylePropertyModulateSelf, Color.FromHex("#753131")),
+
+                //
+                Element<TextureButton>().Class("Refresh")
+                    .Prop(TextureButton.StylePropertyTexture, resCache.GetTexture("/Textures/Interface/Nano/circular_arrow.svg.96dpi.png")),
                 // ---
 
+
```

Comment:
You forgot to delete the extra newline!

```suggestion
```

Other than that, the PR looks good.